types/human-date/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for human-date 1.4
+// Project: https://github.com/montanaflynn/human-date
+// Definitions by: Rico Sandyca Novenza <https://github.com/ricosandyca>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface PrettyPrintOptions {
+    showTime?: boolean;
+}
+
+export interface RelativeTimeOptions {
+    futureSuffix?: string;
+    pastSuffix?: string;
+    presentText?: string;
+    returnObject?: boolean;
+    allUnits?: boolean;
+}
+
+export interface RelativeTimeReturns {
+    seconds?: number;
+    hours?: number;
+    days?: number;
+    years?: number;
+    past?: boolean;
+}
+
+// prettyPrint
+export function prettyPrint(
+    arg: string | Date | number,
+    options?: PrettyPrintOptions
+): string;
+
+// relativeTime
+export function relativeTime(
+    arg: string | Date | number,
+    options?: RelativeTimeOptions
+): string | RelativeTimeReturns;
```

Comment:
Typescript allows methods overloads:
https://www.typescriptlang.org/docs/handbook/functions.html#overloads
so here we have an opportunity to overload the methods with `returnType` and return different result type as per implementation:
```suggestion
export function relativeTime(
    arg: string | Date | number,
    options?: RelativeTimeOptions & { returnObject?: false }
): string;
export function relativeTime(
    arg: string | Date | number,
    options: RelativeTimeOptions & { returnObject: true }
): RelativeTimeReturns;
```

This is optional, you could go with existing one, just let me know.
If you decide to follow that suggestion, then you could also change test details to cover that different overloaded methods now:
```ts
// .relativeTime method
hdate.relativeTime(4); // $ExpectedType string
hdate.relativeTime(4, {futureSuffix: "in the future"}); // $ExpectedType string
hdate.relativeTime("8-16-1987"); // $ExpectedType string
hdate.relativeTime(new Date("8-16-1987")); // $ExpectedType string
hdate.relativeTime(new Date("8-16-1987"), { returnObject: true }); // $ExpectedType RelativeTimeReturns
hdate.relativeTime(75, { allUnits: true }); // $ExpectedType string
```